glide.yaml

```diff
@@ -66,27 +66,23 @@ import:
 - package: github.com/spf13/cobra
   version: f62e98d28ab7ad31d707ba837a966378465c7b57
 - package: k8s.io/gengo
-  version: 9e661e9308f078838e266cca1c673922088c0ea4 # see https://github.com/kubernetes/code-generator/blob/release-1.8/Godeps/Godeps.json
+  version: b58fc7edb82e0c6ffc9b8aef61813c7261b785d4 # in sync with https://github.com/kubernetes/code-generator/blob/kubernetes-1.9.0/Godeps/Godeps.json
   subpackages:
   - args
 - package: github.com/kubernetes/repo-infra
-  version: e26fc85d14a1d3dc25569831acc06919673c545a # latest (Oct 4, 2017)
+  version: 2d2eb5e12b4663fc4d764b5db9daab39334d3f37 # master, latest (Dec 18, 2017)
 - package: k8s.io/client-go
-  version: afb4606c45bae77c4dc2c15291d4d7d6d792196c # release-5.0, latest (Oct 4, 2017)
+  version: kubernetes-1.9.0 # tag/kubernetes-1.9.0
 - package: k8s.io/apimachinery
-  version: 9d38e20d609d27e00d4ec18f7b9db67105a2bde0 # release-1.8, latest (Oct 4, 2017)
+  version: kubernetes-1.9.0 # tag/kubernetes-1.9.0
 - package: k8s.io/kube-openapi
-  version: 868f2f29720b192240e18284659231b440f9cda5 # k8s.io 1.8 (Oct 4, 2017)
+  version: 39a7bf85c140f972372c2a0d1ee40adbf0c8bfe1 # in sync with https://github.com/kubernetes/kubernetes/blob/v1.9.0/Godeps/Godeps.json
 - package: k8s.io/api
-  version: fe29995db37613b9c5b2a647544cf627bfa8d299 # release-1.8, latest (Oct 4, 2017)
+  version: kubernetes-1.9.0 # tag/kubernetes-1.9.0
 - package: k8s.io/apiserver
-  version: f9b476d61dbdf6474e3d42cc5e854d0d8b7f759e # release-1.8, latest (Oct 4, 2017)
+  version: kubernetes-1.9.0 # tag/kubernetes-1.9.0
```

Comment:
Yay, 1.9 versions!

Reply:
Yep, finally we have tags in non-core repos.